cypress/e2e/SubdomainNavigation.cy.ts

```diff
@@ -0,0 +1,221 @@
+import { NFTPageSelector } from '../../src/pages/NFT/selectors';
+import { DomainsPageSelector } from '../../src/pages/Domains/selectors';
+import { SubdomainViewSelector } from '../../src/features/view-subdomains/selectors';
+
+describe('Subdomain Navigation', () => {
+	const rootDomainZna = 'wilder';
+	const subdomainZna = `${rootDomainZna}.snowflake`;
+
+	it('successfully loads correct page for the root domain', () => {
+		cy.visitRootDomain();
+
+		// assert url
```

Comment:
Let me know if you'd prefer these comments to be removed.

Reply:
They can be quite handy - cheers! I think you've done it right in this file, but in general if a line isn't obvious (obvious is like `const firstName = 'hello'; const secondName = 'world'; const fullName = firstname + ' ' + secondName`) a comment can be nice for signposting.